docs/source/admin/traffic_ops.rst

```diff
@@ -0,0 +1,708 @@
+
+:geniso: This object contains configuration options for system ISO generation.
+
+	:iso_root_path: Sets the filesystem path to the root of the ISO generation directory. For default installations, this should usually be set to :file:`/opt/traffic_ops/app/public`.
+
+	.. seealso:: :ref:`tp-tools-generate-iso`
+
+:hypnotoad: This is a group of options used nearly exclusively by the `Legacy Perl Script`_ (as the name implies).
+
+	.. seealso:: For a complete description of this section of the configuration, consult `The Mojolicious Cookbook's section on Hypnotoad <https://mojolicious.org/perldoc/Mojolicious/Guides/Cookbook#Hypnotoad>`_.
+
+	:group:             Sets the UNIX group as which the `Legacy Perl Script`_ will run. By default this should be the ``trafops`` group which is created when the Traffic Ops RPM is installed.
+	:heartbeat_timeout: From the Hypnotoad documentation: "The maximum amount of time in seconds before a worker without a heartbeat will be stopped gracefully".
+	:listen:            This must always be an array containing a single string. This very odd string is apparently a URL. The scheme of the URL specifies the protocol Traffic Ops will use to listen for incoming connections (`traffic_ops_golang`_ ignores that and always uses HTTPS). The host of the URL tells Traffic Ops on what hostname to listen (`traffic_ops_golang`_ understands this as the hostname of the `Legacy Perl Script`_ server), and likewise the port sets the port on which it listens (the port is only used for the `Legacy Perl Script`_ - `traffic_ops_golang`_ uses the setting of ``traffic_ops_golang.port``). The "cert" query parameter sets the location of the SSL certificate to use for encrypting connections, while the "key" query parameter is the certificate's corresponding private key. The default configuration file also has the "verify" query parameter which sets the TLS verification mode - `0` for no verification of certificates, `1` to verify them (`traffic_ops_golang`_ ignores this and uses ``traffic_ops_golang.insecure`` instead). Finally, a list of TLS ciphers to offer to incoming clients of the `Legacy Perl Script`_ delimited by commas can be given in the "ciphers" query parameter.
+	:pid_file:          When the `Legacy Perl Script`_ starts as a daemon, it will write its :abbr:`PID (Process ID)` to this file - so be sure that ``user``/``group`` can properly access it! The default setting is :file:`/var/run/traffic_ops.pid`.
+	:user:              Sets the UNIX user as whom the `Legacy Perl Script`_ will run. By default this should be the ``trafops`` user who is created when the Traffic Ops RPM is installed.
+	:workers:           This is used only by the `Legacy Perl Script`_, and has no known purpose. Though, one could guess that it's the number of concurrent HTTP server workers allowed.
```

Comment:
I believe this is the number of Mojolicious worker processes that will be spawned to handle HTTP requests for TO-Perl.